stubs/docutils/docutils/frontend.pyi

```diff
@@ -81,3 +81,5 @@ class ConfigParser(RawConfigParser):
     def __getattr__(self, name: str) -> Incomplete: ...
 
 class ConfigDeprecationWarning(DeprecationWarning): ...
+
+def get_default_settings(*components) -> Values: ...
```

Comment:
To me `get_default_settings(Parser)` succeeds but `get_default_settings([Parser])` fails  with `AttributeError`, so maybe like this:

```suggestion
def get_default_settings(*components: type[Parser]) -> Values: ...
```

I don't really know anything about docutils though. Maybe a `Parser` class is not the intended thing to pass here, since it is named `components`.

Reply:
Whoops, @Akuli is right, of course. `components` is already a `list`, which then get's passed to `OptionParser()`.

Reply:
Yes, that would agree with my experience that a type hint for `*args` needs to refer to a single element of those `*args`, not to the whole iterable.